r10k.yaml.example

```diff
@@ -1,5 +1,79 @@
-:cachedir: '/var/cache/r10k'
-:sources:
-  :plops:
-    remote: 'git@github.com:my-org/org-shared-modules'
-    basedir: '/etc/puppet/environments'
+---
+
+# The cachedir setting controls where cached content, such as mirrored Git
+# repositories, are stored. This location should be persistent, as environments
+# and modules may rely on these files in order to be updated.
+#
+# See https://github.com/puppetlabs/r10k/blob/master/doc/git/cloning-and-mirroring.mkd
+# for more information on how r10k caches Git repositories.
+#
+# The default value is "~/.r10k"
+cachedir: '/var/cache/r10k'
+
+# The sources setting determines what repositories r10k will use when creating
+# Puppet environments. The structure is a map of source names and a map of the
+# source settings.
+sources:
+  # Each source should have a unique name, and preferrably should only use
+  # alphanumeric characters and underscores.
+  operations:
+
+    # The 'remote' setting determines the location of the VCS repository used
+    # for creating environments.
+    remote: 'git@github.com:my-org/org-operations-modules'
+
+    # The 'basedir' setting specifies where environments will be created for
+    # this source. This setting should match the Puppet 'environmentpath'
+    # setting in puppet.conf.
+    #
+    # See http://docs.puppetlabs.com/references/3.7.latest/configuration.html#environmentpath
```

Comment:
I've updated this to point to the `/latest/` uri; it'll stay up to date until the semantics of the setting changes, at which point we'll need to change this setting regardless.